ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -71,7 +71,7 @@ tests:
     cluster_profile: aws-centos-40
 - as: e2e-etcd-quorum-loss
   commands: |
-    recover-from-etcd-quorum-loss
+    recover-from-etcd-quorum-loss  
     TEST_SUITE=openshift/conformance/parallel run-tests
```

Comment:
trailing whitespace?

Reply:
I need it to trigger the test - this commit would be removed later

Reply:
OK gotcha I find this a nice little trick `git commit --amend --no-edit` :)